### ISSUE TITLE
Make ReleaseCmd add a git tag

### ISSUE THREAD

**Issue by Stebalien** [whyrusleeping/gx#157]
Specifically, I'd tag it with:

```bash
git tag -a "v$VERSION" -m "gx release published as $HASH"
```

To do this, we'll need:

* Access to the hash.
* The ability to run multiple commands (preferably without bash).
* Decide on whether or not we want to sign the tags.
* Changelogs?

**Comment by whyrusleeping** [comment]
Yeah, I really want this. @frrist proposed this back in december I remember.

**Comment by dignifiedquire** [comment]
This is even more important now that gomod is a thing, as it strongly relies on git tags for version information.
